sys/ztimer/util.c

```diff
@@ -55,14 +55,13 @@ void ztimer_sleep(ztimer_clock_t *clock, uint32_t duration)
     mutex_lock(&mutex);
 }
 
-void ztimer_periodic_wakeup(ztimer_clock_t *clock, ztimer_now_t *last_wakeup,
+void ztimer_periodic_wakeup(ztimer_clock_t *clock, uint32_t *last_wakeup,
```

Comment:
Needs to be changed in the header file as well

Reply:
It actually is already in master :confused:.